src/Scientist/Internals/Experiment.cs

```diff
@@ -10,7 +10,25 @@ internal class Experiment<T, TClean> : IExperiment<T, TClean>, IExperimentAsync<
 
         private static readonly Func<Task<bool>> _alwaysRun = () => Task.FromResult(true);
         private static readonly Action<Operation, Exception> _alwaysThrow
-            = (operation, exception) => { throw exception; };
+            = (operation, exception) =>
+            {
+                string exceptionMessage;
+                switch (operation)
```

Comment:
Agreed - I think if we add another `Operation` to the list then we should be using it explicitly, and it should be known.